cpp/util/etcd.cc

```diff
@@ -980,7 +980,8 @@ list<EtcdClient::HostPortPair> SplitHosts(const string& hosts_string) {
   list<EtcdClient::HostPortPair> ret;
   for (const auto& h : hosts) {
     vector<string> hp(util::split(h, ':'));
-    CHECK_EQ(2, hp.size()) << "Invalid host:port string: '" << h << "'";
+    CHECK_EQ(static_cast<size_t>(2), hp.size()) << "Invalid host:port string: '"
```

Comment:
Since this is for a constant, it should be fine (the compiler will do some promotion to make it work), it's the signedness that's a problem (although I'll say that's some crappy static analysis that gives you a warning for comparing a signed with an unsigned, when the "signed" one is a positive constant!).